index.yaml

```diff
@@ -3168,13 +3170,27 @@ entries:
       catalog.cattle.io/release-name: rancher-prom2teams
     apiVersion: v1
     appVersion: 3.2.1
-    created: "2021-06-22T16:00:34.843943-07:00"
+    created: "2021-06-22T16:43:31.595226-07:00"
+    description: A Helm chart for Prom2Teams based on the upstream https://github.com/idealista/prom2teams
+    digest: 0df8093068762ff76d4fc6b82005cac4992326721541995529645d854a3b417e
+    name: rancher-prom2teams
+    urls:
+    - assets/rancher-prom2teams/rancher-prom2teams-0.2.100+up0.2.0.tgz
+    version: 0.2.100+up0.2.0
+  - annotations:
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/hidden: "true"
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/release-name: rancher-prom2teams
+    apiVersion: v1
+    appVersion: 3.2.1
+    created: "2021-06-22T16:43:31.594721-07:00"
     description: A Helm chart for Prom2Teams based on the upstream https://github.com/idealista/prom2teams
-    digest: a8c8b68f1f56ef315ef5e44d3bab1cb9dcc24621f8e3c253738356fa7555ba79
+    digest: 2c83714457d157bc77c0f74ab01c1db84b63c5a811a99225b626f297455e0e3b
     name: rancher-prom2teams
     urls:
-    - assets/rancher-prom2teams/rancher-prom2teams-0.2.0.tgz
-    version: 0.2.0
+    - assets/rancher-prom2teams/rancher-prom2teams-0.2.000.tgz
+    version: 0.2.000
```

Comment:
This is an expected change. 
`0.2.000` is the released version in release-v2.5. `0.2.0` was created during developing in the dev-v2.5 branch sometime in the past.